src/app/Pages/Browser/AWS.tsx

```diff
@@ -33,7 +33,7 @@ const AWS: React.FunctionComponent<{title: string}> = ({title}) => {
           <Title headingLevel="h1"  size={TitleSizes['4xl']}>AWS Image Browser</Title>
         </Bullseye>
       </PageSection>
-      <ImageDataTable tableColumns={columns} />
+      <ImageDataTable tableColumns={columns} pathPrefix={'https://poc.imagedirectory.cloud/images/v1/idx/list/sort-by-date-aws'}/>
```

Comment:
Ah - do we need to remove the `poc` sub-domain at all places? :thinking:

Reply:
Jap! ONce the infrastructure changes are in place (redirect without the poc subdomain) we can remove all of the tags ;)

Reply:
I hope it is this simple: https://github.com/redhatcloudx/infrastructure/pull/29